src/main/app/src/app/klanten/model/klanten/klant-gegevens.ts

```diff
@@ -8,4 +8,5 @@ export class KlantGegevens {
 
     klant: Klant;
     betrokkeneRoltype?: Roltype;
+    betrokkeneToelichting?: string;
```

Comment:
Die klant attribuut is niet nodig. Zie bijvoorbeeld DocumentOntkoppelGegevens.ts. 
Ik bedoel dat je alleen maar public hoeft te zetten voor het attribut in de constructor. En dan kan ook die assignment uit de construcor

En we doen geen ? bij attributen.

Reply:
Was ik even vergeten, die constructor syntax